samples/WebUSB.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2017, Intel Corporation.
+
+console.log("Starting WebUSB sample...");
+
+var webusb = require('webusb');
+
+// On a Linux PC with the Google Chrome >= 60 running, when you plug the device
+//   in via USB, Chrome should detect it and display a notification prompting
+//   you to visit the URL specified below.
+// Note: You should only use https URLs because Chrome will not display a
+//   notification for other URLs.
+
+// suggest that the browser visit this URL to connect to the device
+webusb.setURL("https://github.com/01org/zephyrjs");
```

Comment:
I think you mean "https://github.com/01org/zephyr.js".

Reply:
Perhaps the comment is misleading, but to connect to the device you should go to https://01org.github.io/zephyrjs-ide/

But if what you meant was you want them to go to our github to learn about connecting, that makes sense too.